src/validate.ts

```diff
@@ -38,7 +32,7 @@ function validate(toolOutput: any[]): boolean {
     /*
      * Check #1: vertices are emitted before connecting edges
      */
-    process.stdout.write(`${outputMessage}\r`);
+    console.log(`${outputMessage}`);
```

Comment:
Any special reason for this change?

I used `process.stdout.write` in order to be able to write over it afterwards. Do you know a better way to change the line that has been printed to the console already?

Reply:
I see...

But I do not find where the console get rewrite.

https://stackoverflow.com/a/37561178

Reply:
Hi @jumattos, I just revert the change since it's out of the scope in this PR.

Reply:
It's my first time meddling with the console. Thanks for this link, it points a much better way of doing things than my current approach. I will change it in another PR.

I will go ahead and approve/merge this one. Thanks for the contribution! 😄